Src/IronPython/Compiler/Ast/PythonNameBinder.cs

```diff
@@ -385,6 +385,16 @@ class PythonNameBinder : PythonWalker {
         // CallExpression
         public override bool Walk(CallExpression node) {
             node.Parent = _currentScope;
+
```

Comment:
The method should be moved out of the generated block and `generate_walker.py` should be updated accordingly.